spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/transfer/DataRequest.java

```diff
@@ -47,6 +47,8 @@
 
     private TransferType transferType;
 
+    private boolean isSyncRequest = false;
```

Comment:
Would remove the `is` prefix on the field name, and also the `Request` suffix is not needed, since the class is already called `DataRequest`, I would call it just `sync`.